src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -34,7 +34,10 @@ interface ITraitChangeEvent {
 	browserEvent?: UIEvent;
 }
 
-type ITraitTemplateData = HTMLElement;
+type ITraitTemplateData = {
+	container: HTMLElement;
+	disposables?: DisposableStore;
```

Comment:
Should this just go into a separate `IAccessibilityTemplateData` type, and not be optional? We can then leave this type as is and have a much more concise change.

Reply:
sounds like that would help with the current typing errors